src/style.css

```
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 300;
+  src: local("Avenir"),
+    url("fonts/Avenir.ttc") format("ttc")
+    url(fonts/Avenir-Book.ttf) format("ttf")
+    url(fonts/Avenir-Heavy.ttf) format("ttf");
+}
+
+.search-bar2 {
+  width: 100%;
+  margin-top: 20px;
+  line-height: 19px;
+}
+
+.search-bar {
+  width: 100%;
+  margin-top: 20px;
+  line-height: 22px;
+}
+
+.label_class1 {
+  position: relative;
+  display: block;
+}
+
+.search_first {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-size: 16px;
+}
+
+.icon_first {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+}
+
+::placeholder {
+  font-family: "Avenir", "Arial", sans-serif;
+  font-weight: 300;
+}
+
+.label_class2 {
+  position: relative;
+  display: block;
+}
+
+.search_second {
+  width: 100%;
+  height: 42px;
+  padding-left: 33px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-size: 14px;
+}
+
+.icon_second {
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+input:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+  font-weight: 900;
+}
```

To remove bold border on focus
```suggestion
  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
  font-weight: 900;
  outline: none;
}
```